_includes/documentation.html

```diff
@@ -6,7 +6,7 @@
                 <div class="col-lg-4 col-md-6 col-sm-6 cardlist-col">
                   <div class="cardlist-card doc-card">
                     <div class="card-heading">
-                      <h4>{{card[0]}}</h4>
+                      <h2>{{card[0]}}</h2>
```

Comment:
I don't understand this change. Is the rationale to preserve semantic header hierarchy?

Reply:
Yes, this is recommended by Lighthouse. Here is the explanation they offer. https://web.dev/heading-order/?utm_source=lighthouse&utm_medium=devtools